src/Components/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  title: string,
+  handleNewTitle: (event: React.ChangeEvent<HTMLInputElement>) => void,
+  addNewTodo: (title: string) => void,
+  isDisabledInput: boolean,
+};
+
+export const Header: React.FC<Props> = ({
+  todos,
+  title,
+  handleNewTitle,
+  addNewTodo,
+  isDisabledInput,
+}) => {
+  const isActive = todos.filter(todo => !todo.completed);
```

Comment:
actually there are a lot of duplicate of this filter. consider moving it into a separated helper function
```suggestion
  const activeTodos = todos.filter(todo => !todo.completed);
```